python/batch-predictor/docker/app.Dockerfile

```diff
@@ -25,5 +25,6 @@ RUN if [[ ! -z "$GOOGLE_APPLICATION_CREDENTIALS" ]]; then gcloud auth activate-s
 RUN gsutil -m cp -r ${MODEL_URL} .
 RUN /bin/bash -c ". activate ${CONDA_ENVIRONMENT} && \
     sed -i 's/mlflow[><=]\{0,1\}.*$/mlflow==1.26.1/g' ${HOME}/model/conda.yaml && \
+    sed -i '/- merlin-sdk[><=]\{0,1\}.*$/d' ${HOME}/model/conda.yaml && \
```

Comment:
can it just be 

`sed -i '/merlin-sdk/d' ${HOME}/model/conda.yaml`?

Reply:
Oh yes, it's much simpler.